### DISCUSSION TITLE
Welcome to ametrine.host Discussions!

### DISCUSSION THREAD

**Discussion by Em1tt** [Em1tt/amethyst.host#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Em1tt** [comment]
Trollge

**Comment by ghost** [comment]
sorry what is amethyst.host?

**Reply by Em1tt** [reply]
Fixed
